src/Stripe.net/Infrastructure/JsonConverters/SourceConverter.cs

```diff
@@ -29,6 +29,12 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
                 Id = incoming.SelectToken("id").ToString()
             };
 
+            if (incoming.SelectToken("object")?.ToString() == "account")
+            {
+                source.Type = SourceType.Account;
+                source.Account = Mapper<StripeAccount>.MapFromJson(incoming.ToString());
+            }
+
```

Comment:
So, I wanted to do an `if/else if/ else if/ ... / else` because it's faster but seems there's only one in the entire lib so wasn't sure if there was a reason.

Reply:
I don't think there's any specific reason -- these cases are all mutually exclusive so using `else if` would be fine.